### DISCUSSION TITLE
Icon request: api

### DISCUSSION THREAD

**Discussion by Valerzen88** [FortAwesome/Font-Awesome#21120] [Icon Requests]
### What can it be used for?

I need an API-icon for software website.

### Example image (optional)

Something like { API } or the same in a gear.
https://waminox.com/data/api-a397cc184c5622fb5130af1b7baf149d-1170x658.png
https://st3.depositphotos.com/14846838/18296/v/1600/depositphotos_182963760-stock-illustration-api-interface-vector-icon.jpg

### Icon request checklist

- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have included at least one example image if it could be helpful (optional)
- [X] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Is this a brand?

**Comment by Valerzen88** [comment]
Hi!

No, it should just symbolize an API (application programming interface).

Thx

**Comment by dolibarr95** [comment]
+1

**Comment by Pezhvak** [comment]
+1

**Comment by mattroche** [comment]
Ask your developers.  They have to know what this is.  
We are trying to use the icons to show how other services are talking to ours, and we need:
API
Webhook
SFTP
File transfer

All pretty common in software development.  None are brands, all are protocols or approaches

**Comment by engineertdog** [comment]
+1 any update?

**Comment by elier** [comment]
+1

**Comment by cyrlop** [comment]
+1

**Comment by ghost** [comment]
+1.

Surprising that this one isn't available already.

I'd suggest something like this:

https://www.vectorstock.com/royalty-free-vector/black-cloud-api-interface-icon-isolated-vector-26101410

**Comment by tlbignerd** [comment]
I've been expecting something like this from v5. I end up using gears or something, but would prefer a dedicated icon showing how information is available to external users. @DM2489 suggestion is probably the closest I've seen.

**Comment by LunaMooncraft** [comment]
+1

**Comment by cary-hu** [comment]
+1

**Comment by TerryDieckmann** [comment]
+1

**Comment by OscarBeiro** [comment]
+1

**Comment by maewCP** [comment]
+1

**Comment by l-Legacy-l** [comment]
+1

**Comment by h2im** [comment]
We need "API"

**Comment by thomas-bios** [comment]
+1, still nothing in 2024?

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Added to 7.2: https://fontawesome.com/search?q=api&ip=classic&s=solid&c=connectivity
